website/docs/reference/terraform.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Using Unleash through Terraform
+---
+
+:::note Availability
+
+**Version**: `5.6+`
+
+:::
+
+
+## Overview
+
+The [Unleash Terraform provider](https://github.com/Unleash/terraform-provider-unleash) allows you to manage and configure your Unleash programmatically using Terraform, enabling automated and scalable management for your infrastructure.
+By integrating Unleash with Terraform, you can use infrastructure as code (IaC) to optimize the setup and maintenance of your Unleash instance configuration.
+
+The Unleash Terraform provider is designed to help you with the **initial setup and configuration** of an instance. The provider does not support managing feature flags through Terraform. Since most [feature flags are short-lived](/topics/feature-flags/feature-flag-best-practices#7-make-flags-short-lived), we recommend managing them through the Unleash Admin UI.
+
+For a detailed tutorial, follow along with [Managing Unleash through Terraform](https://www.youtube.com/watch?v=B4OIBC1u1ns).
+For more examples for the specific resources and data sources, visit the [Terraform registry](https://registry.terraform.io/providers/Unleash/unleash/latest/docs/data-sources/permission).
+
+## Manage Terraform access
+
+The permissions of the API token you use with Terraform and your Unleash [plan](https://www.getunleash.io/pricing) determine what objects Terraform can manage.
+- For Open-Source and Pro, use an [admin token](/reference/api-tokens-and-client-keys#admin-tokens) or [personal access tokens](/reference/api-tokens-and-client-keys#personal-access-tokens).
```

Comment:
We're trying to move people away from using admin tokens, I think the feature is even disabled in the UI, so I'd just keep PAT for OSS